addons/website_event_track/models/event_track.py

```diff
@@ -58,11 +58,11 @@ def _get_default_stage_id(self):
     name = fields.Char('Title', required=True, translate=True)
     active = fields.Boolean(default=True)
     user_id = fields.Many2one('res.users', 'Responsible', tracking=True, default=lambda self: self.env.user)
-    partner_id = fields.Many2one('res.partner', 'Speaker')
-    partner_name = fields.Char('Speaker Name')
-    partner_email = fields.Char('Speaker Email')
-    partner_phone = fields.Char('Speaker Phone')
-    partner_biography = fields.Html('Speaker Biography')
+    partner_id = fields.Many2one('res.partner', 'Company')
```

Comment:
What ? Company ? in backend ?

Reply:
well it's just a name and it is apparently clearer for sbu

Reply:
I jdiscuss with him.

But what is your own opinion on this change ? 
You have your voice when idea seems wrong ;)

Reply:
in fact I get it, it's not ok to do this

Reply:
My opinion is that the old name was not the best name in order to be clear but Company was not a problem for me as it can be clearer in the form

Reply:
But indeed with your explanations there is something wrong with 'Company'